src/BinaryRewriting/BclRewriter/project.json

```diff
@@ -0,0 +1,16 @@
+﻿{
+  "dependencies": {
+    "Microsoft.Cci": "4.0.0-beta-23321",
+    "System.Console": "4.0.0-beta-23321",
+    "System.Xml.ReaderWriter": "4.0.10",
+    "System.Linq": "4.0.0-beta-23321",
+    "System.Reflection.TypeExtensions": "4.0.0-beta-23321",
+    "Microsoft.NETCore.Runtime.CoreCLR": "1.0.0-beta-23127",
```

Comment:
You are right, I should reference System.Reflection.Extensions instead. I just pushed a change that fixes this. Thanks!